### DISCUSSION TITLE
六分街录像店

### DISCUSSION THREAD

**Discussion by krunsa111** [share121/inter-knot#4747] [绝区零]
就我觉得录像店那对兄妹关系不太正常吗
那哥哥对妹妹有点太爱了（我没有羡慕）
![IMG_20240910_191141](https://github.com/user-attachments/assets/17548bb0-4806-4570-826e-0c2b8fffc9a6)

**Comment by fuqikilroy** [comment]
不正常？
两个不像亲兄妹，像伙伴

**Comment by Maridizk** [comment]
可是，哥哥的身子太香了
